test-infra/scripts/README.md

```diff
@@ -0,0 +1,356 @@
+# Helper scripts
+
+This directory contains helper scripts used by Prow test jobs, as well as local
+development scripts.
+
+## Using the `presubmit-tests.sh` helper script
+
+This is a helper script to run the presubmit tests. To use it:
+
+1. Source this script.
+
+1. [optional] Define the function `build_tests()`. If you don't define this
+   function, the default action for running the build tests is to:
+
+   - check markdown files
+   - run `go build` on the entire repo
+   - run `/hack/verify-codegen.sh` (if it exists)
```

Comment:
no `hack/verify-codegen.sh` in this PR.

Reply:
This is not yet complete, but the `presubmit-test` contains calls to the unit and build tests. e2e-tests are the actuall end-to-end tests. You could ommit the e2e-tests script, in which case no e2e tests are run. The unit tests are run automatically. 

But let me complete the template scripts which can be taken over directly.

I will also add build.sh to the template so that this can be reused for plugins.

Unfortunately I got an urgent task for today, so could be that I continue on this PR only tomorrow ;-(